pallets/authorship/src/tests.rs

```diff
@@ -0,0 +1,59 @@
+use crate::{mock::*,ExtrinsicAuthors,Error};
+use cord_primitives::AccountId;
+use frame_support::{assert_ok,assert_err, error::BadOrigin};
+use frame_system::RawOrigin;
+
+
+
+#[test]
+fn add() {
+	ExtBuilder::default().build().execute_with(|| {
+		let test_mock_authors: Vec<CordAccountOf<Test>> = vec![TEST_AUTHOR2,TEST_AUTHOR3];
+		assert_ok!(Authorship::add(
+			RawOrigin::Root.into(),
+			test_mock_authors.clone()
+		));
```

Comment:
This call should have failed. We need to add an error condition to the lib, to check for duplicates